repositories/src/main/scala/cromwell/pipeline/datastorage/formatters/UserFormatters.scala

```diff
@@ -0,0 +1,20 @@
+package cromwell.pipeline.datastorage.formatters
+
+import cats.implicits.catsStdShowForString
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.model.wrapper.{ Name, Password, UserEmail, UserId }
+import play.api.libs.json.{ Format, Json, OFormat }
+
+object UserFormatters {
+  implicit lazy val userFormat: OFormat[User] = Json.format[User]
+  implicit lazy val profilePictureFormat: OFormat[ProfilePicture] = Json.format[ProfilePicture]
+  implicit val UserNoCredentialsFormat: OFormat[UserNoCredentials] =
+    Json.format[UserNoCredentials]
+  implicit lazy val updatePasswordRequestFormat: OFormat[PasswordUpdateRequest] = Json.format[PasswordUpdateRequest]
+  implicit lazy val updateRequestFormat: OFormat[UserUpdateRequest] = Json.format[UserUpdateRequest]
+
+  implicit lazy val nameFormat: Format[Name] = Name.wrapperFormat
+  implicit lazy val passwordFormat: Format[Password] = Password.wrapperFormat
+  implicit lazy val userEmailFormat: Format[UserEmail] = UserEmail.wrapperFormat
+  implicit lazy val userIdFormat: Format[UserId] = UserId.wrapperFormat
```

Comment:
This is cheating :)
`wrapperFormat` must also be moved from `Wrapped` object into something like `WrappedFormatters` (maybe this is a bad name and there is a better one). You can put all formatters for these wrappers into that object.